src/components/common/codeInput/CodeInput.tsx

```diff
@@ -115,7 +116,7 @@ const CodeInput = ({
     >
       Custom Code
     </MenuItem>,
-    valueSets?.map((vs) => {
+    _.sortBy(valueSets, "display_name")?.map((vs) => {
```

Comment:
We also need to sort `codeSystemMenuOptions` in this component

Reply:
It felt like we need `codeMenuOptions` to also be sorted, but not sure if we need to sort them based on codes or based on display_name. Did we get a chance to check with POs ?